spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-lattice/src/main/java/org/springframework/cloud/data/module/deployer/lattice/LrpModuleDeployer.java

```diff
@@ -66,10 +67,11 @@ public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		List<EnvironmentVariable> environmentVariables = new ArrayList<EnvironmentVariable>();
 		Collections.addAll(environmentVariables, lrp.getEnv());
 		environmentVariables.add(new EnvironmentVariable("MODULES", request.getCoordinates().toString()));
-		for (Map.Entry<String, String> entry : request.getDefinition().getParameters().entrySet()) {
-			environmentVariables.add(new EnvironmentVariable(entry.getKey(), entry.getValue()));
-		}
-		for (Map.Entry<String, String> entry: request.getDeploymentProperties().entrySet()) {
+		Map<String, String> rawArgs = new HashMap<>();
+		rawArgs.putAll(request.getDefinition().getParameters());
+		rawArgs.putAll(request.getDeploymentProperties());
+		Map<String, String> transformed = ModuleLauncherArgumentsHelper.qualifyArgs(0, rawArgs);
```

Comment:
change "transformed" to "qualifiedArgs"? (the code is easier to read when terminology is consistent, rather than multiple terms being introduced for the same thing)